webhooks-extension/src/components/WebhookBranches/WebhookBranches.js

```diff
@@ -0,0 +1,151 @@
+import React, { Component } from "react";
```

Comment:
single quotes for consistency with the rest of the code
can you also open an issue to track enabling prettier on this project to avoid this type of code style issue, or we may already have one

Reply:
weird since this is a new file to the repo I applied prettier before pushing... thought all the quotes would be fixed automatically like it does in the dashboard repo. Is it possible that its not configured properly in this repo?

Reply:
I don't see a .prettierrc in the webhooks-extension root, you could add one with this content (same as dashboard):

```
{
  "singleQuote": true,
  "trailingComma": "none"
}
```